### PR TITLE
Update BA18 Implement KS06 and AS16

### PR DESCRIPTION
**Pull request by glavrentiadis**
Remove testing plot from BA18
  Implement Kempton Stewart (2006) and Afshari and Stewart (2016) duration models

### CODE REVIEW

File: pygmm/bayless_abrahamson_2019.py [hunk 1]
```diff
@@ -165,7 +165,8 @@ def _calc_site_response(self, ln_eas_ref: float) -> np.ndarray:
 
         f_s = f_sl + f_nl
 
-        if v_s30 < 800:
+        flag_test = False
```
Comment (arkottke) [inline pygmm/bayless_abrahamson_2019.py:168]: Is this needed?

Comment (glavrentiadis) [reply pygmm/bayless_abrahamson_2019.py:168]: The condition `v_s30 < 800` was in the original script. I added `flag_test = False` to suppress the plotting if it is not needed. Alternatively, lines 168 to 172 can be deleted or kept in as is in the main branch.

File: pygmm/kempton_stewart_2006.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        
+        #path duration
+        f_1 = np.array([0.07, 0.15, 0.10, 0.15]) * s.dist_rup
+        
+        #site duration
+        f_2 = (np.array([0.82, 3.00, 1.40, 3.99]) + 
+               np.array([-0.0013,-0.0041,-0.0022,-0.0062]) * s.v_s30 )
+        
+        #total druation
+        self._ln_dur = np.log( f_0 + f_1 + f_2 )
```
Comment (arkottke) [inline pygmm/kempton_stewart_2006.py:55]: I might turn this into a pd.Series with names so that it more easily accessible.

**Comment by arkottke** [comment]
Do you have any test cases for these?

**Comment by glavrentiadis** [comment]
No, I haven't written any test cases.

**Pull request merged**